common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -87,13 +87,22 @@ class CelebornConf(loadDefaults: Boolean) extends Cloneable with Logging with Se
     this
   }
 
+  private def warnIfInternalTransportModule(module: String, key: String): Unit = {
+    // for now, assert false to test
```

Comment:
Could this add the TODO comment for this?

Reply:
Should remove it - I had an assertion here earlier :-)